packages/twenty-front/src/modules/ui/layout/dropdown/components/Dropdown.tsx

```diff
@@ -63,6 +65,10 @@ export const Dropdown = ({
 
   const { isDropdownOpen, toggleDropdown, closeDropdown, dropdownWidth } =
     useDropdown(dropdownId);
+
+  const { handleResetSelectedPosition } = useSelectableList(
+    SINGLE_ENTITY_SELECT_BASE_LIST,
+  );
```

Comment:
I'm not sure if this is a good idea to include this selectable list hook here, I had to do this because in `RecordDetailRelationSection`, we use `Dropdown` which already has a `KeyPress.Escape` handler, but we also use  `SingleEntitySelectMenuItemsWithSearch` as dropdownComponents which also has has a `KeyPress.Escape` handler. 

Not sure what I can do here, any suggestions are welcome 👍